action/transfer.go

```diff
@@ -79,8 +79,8 @@ func (tsf *Transfer) TotalSize() uint32 {
 	if tsf.amount != nil && len(tsf.amount.Bytes()) > 0 {
 		size += uint32(len(tsf.amount.Bytes()))
 	}
-
-	return size + uint32(len(tsf.payload))
+	// 65 is the pubkey size
+	return size + uint32(len(tsf.payload)) + 65
```

Comment:
verified sm2 signature/tx won't be accepted (due to wrong length), so pubkey size = constant 65